pdns/minicurl.cc

```diff
@@ -130,9 +143,17 @@ void MiniCurl::setupURL(const std::string& str, const ComboAddress* rem, const C
   d_byteslimit = byteslimit;
   if (d_byteslimit > 0) {
     /* enable progress meter */
+#if defined(LIBCURL_VERSION_NUM)
+// CURL_AT_LEAST_VERSION has been introduced AFTER 7.32.0
```

Comment:
what's the relation between the `#if` and the comment?

Reply:
CURL_AT_LEAST_VERSION is easier to read than LIBCURL_VERSION_NUM

Reply:
Right! Do we even need to check if LIBCURL_VERSION_NUM is defined? (if not, you can ignore my other suggestion)

Reply:
I don't think so because we expect curl functions to be defined. But it is also checked at various other places so I kept the same pattern.

Reply:
It's confusing that `CURLOPT_NOPROGRESS` depends on whether `LIBCURL_VERSION_NUM` is defined. Maybe move the defined() check to go with the version check, like in the header?

Reply:
When you are testing a preprocessor symbol for greater than zero a `defined` guard is not _needed_, as undefined symbols evaluate to 0. Still I think it is worthwhile to have an explicit `defined` guard, as testing for other conditions (e.g. `== 0`) need it.